bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,403 @@
+
+    def __init__(self, bot: SeasonalBot):
+        self.bot = bot
+        self.http_session: ClientSession = bot.http_session
+
+        self.refresh_genres_task.start()
+
+    @tasks.loop(hours=1.0)
+    async def refresh_genres_task(self) -> None:
+        """Refresh genres in every hour."""
+        try:
+            await self._get_genres()
+        except Exception as e:
+            logger.warning(f"There was error while refreshing genres: {e}")
+            return
+        logger.info("Successfully refreshed genres.")
+
+    async def _get_genres(self) -> None:
+        """Create genres variable for games command."""
+        body = "fields name; limit 100;"
+        async with self.http_session.get(f"{BASE_URL}/genres", data=body, headers=HEADERS) as resp:
+            result = await resp.json()
+
+        genres = {genre["name"].capitalize(): genre["id"] for genre in result}
+
+        self.genres = {}
+
+        # Replace complex names with names from ALIASES
+        for genre_name, genre in genres.items():
+            if genre_name in ALIASES:
+                for alias in ALIASES[genre_name]:
+                    self.genres[alias] = genre
+            else:
+                self.genres[genre_name] = genre
+
+    @group(name="games", aliases=["game"], invoke_without_command=True)
+    async def games(self, ctx: Context, genre: Optional[str] = None, amount: int = 5) -> None:
+        """
+        Get random game(s) by genre from IGDB. Use .games genres command to get all available genres.
+
+        Also support amount parameter, what max is 25 and min 1, default 5. Use quotes ("") for genres with multiple
+        words.
+        """
+        # When user didn't specified genre, send help message
+        if genre is None:
+            await ctx.send_help("games")
+            return
+
+        # Capitalize genre for check
+        genre = genre.capitalize()
+
+        # Check for amounts, max is 25 and min 1
+        if not 1 <= amount <= 25:
+            await ctx.send("Your provided amount is out of range. Our minimum is 1 and maximum 25.")
+            return
+
+        # Get games listing, if genre don't exist, show error message with possibilities.
+        # Offset must be random, due otherwise we will get always same result (offset show in which position should
+        # API start returning result)
+        try:
+            games = await self.get_games_list(amount, self.genres[genre], offset=random.randint(0, 150))
+        except KeyError:
+            possibilities = "`, `".join(difflib.get_close_matches(genre, self.genres))
+            await ctx.send(f"Invalid genre `{genre}`. {f'Maybe you meant `{possibilities}`?' if possibilities else ''}")
+            return
+
+        # Create pages and paginate
+        pages = [await self.create_page(game) for game in games]
+
+        await ImagePaginator.paginate(pages, ctx, Embed(title=f"Random {genre} Games"))
```

Comment:
Right now there are way too many blank lines between games, consider using the keyword `empty`

![image](https://user-images.githubusercontent.com/19921765/75874563-8fb4cf80-5e44-11ea-83ef-7f80f91708fe.png)

Thanks @sco1 for finding out about that!